.github/workflows/weekly.yml

```diff
@@ -35,15 +35,15 @@ jobs:
         }, {
           flink: 1.18-SNAPSHOT,
           branch: main
+        }, {
+          flink: 1.19-SNAPSHOT,
+          branch: main
         }, {
           flink: 1.16.2,
           branch: v1.0
         }, {
           flink: 1.17.1,
           branch: v1.0
-        }, {
-          flink: 1.18-SNAPSHOT,
-          branch: v1.0
         }]
```

Comment:
I think we should't remove this lines here.

```js
        {
          flink: 1.16.2,
          branch: v1.0
        }, {
          flink: 1.17.1,
          branch: v1.0
        }, {
          flink: 1.18.0,
          branch: v1.0
        }, {
          flink: 1.19-SNAPSHOT,
          branch: v1.0
        }
```

Reply:
Does version 1.0 support 1.18 and 1.19?

Reply:
For 1.19 we need pin the version in pom.xml.
I think we can cherry-pick this commit into v1.0 branch as well.